warehouse/query-core/src/main/java/datawave/query/jexl/visitors/BoundedRangeDetectionVisitor.java

```diff
@@ -81,7 +85,32 @@ public Object visit(ASTNRNode node, Object data) {
         }
         
         return false;
-        
+    }
+    
+    // ensure we short circuit these nodes
```

Comment:
Add a brief comment explaining why these nodes were chosen, something like "short circuit on nodes that make up a bounded range"

Reply:
Done